src/ui/server/controllers/insurance/policy/multiple-contract-policy/index.ts

```diff
@@ -123,6 +132,8 @@ export const post = async (req: Request, res: Response) => {
     return res.redirect(PROBLEM_WITH_SERVICE);
   }
 
+  const { policy } = application;
```

Comment:
do we need to have a check where if no policy, then it goes to PROBLEM_WITH_SERVICE

Reply:
No, `policy` object will always exist; It's created when the application is created.

If an application does not exist, server middleware will redirect the user before it reaches this controller 👍